Runtime/AvatarCreator/Scripts/UI/Elements/AvatarListElement.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.Serialization;
+using UnityEngine.UIElements;
+using Button = UnityEngine.UI.Button;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    public class AvatarListElement : MonoBehaviour
+    {
+        private enum AvatarListFilter
+        {
+            Application,
+            None
+        }
+
+        private const string TAG = nameof(AvatarListElement);
+
+        [SerializeField] private Transform avatarsContainer;
+        [FormerlySerializedAs("avatarListButton")]
+        [SerializeField] private AvatarListItem avatarListItem;
+        [SerializeField] private AvatarListFilter avatarListFilter;
+        public UnityEvent<string> onAvatarSelect;
+        public UnityEvent<string> onAvatarModify;
+        public UnityEvent<string> onAvatarDeletionStarted;
+
+        private AvatarAPIRequests avatarAPIRequests;
+
+        private readonly Dictionary<string, AvatarListItem> avatarListItems = new Dictionary<string, AvatarListItem>();
+
+        public async void LoadAndCreateUserAvatars()
+        {
+            if (!AuthManager.IsSignedIn && !AuthManager.IsSignedInAnonymously)
+            {
+                SDKLogger.Log(TAG, "Not signed in");
+                return;
+            }
+
+            avatarAPIRequests ??= new AvatarAPIRequests();
+
+            var avatarPartnerArr = await avatarAPIRequests.GetUserAvatars(AuthManager.UserSession.Id);
```

Comment:
Minor naming suggestion, since it a dictionary I suggest maybe following the convention of valueByKey for naming. 
EG partnerByAvatarId or something like that

```suggestion
            var partnerByAvatarId = await avatarAPIRequests.GetUserAvatars(AuthManager.UserSession.Id);
```
Kinda similar to what we have in SelectionElement 
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/6257d8e4-6bbc-4f7e-a977-f7877145c53b)